samples/bot-conversation/nodejs/bots/teamsConversationBot.js

```diff
@@ -70,6 +70,26 @@ class TeamsConversationBot extends TeamsActivityHandler {
                 // Save information about the sent message and its ID (resourceResponse.id).
             }));
         });
+
+        // This method is invoked when message sent by user is updated in chat.
+        this.onTeamsMessageEditEvent(async (context, next) => {
```

Comment:
Technically, the `this.onTeamsMessageEditEvent` method is not "invoked". Instead it registers the lambda function which will then be invoked when the event occurs.

Reply:
`This method registers the lambda function, which will be invoked when message sent by user is updated in chat`. I think this is a better method description.

Reply:
For further clarification, In contrary to JS, the C# equivalent method is in fact invoked.

Reply:
yes changed the descriptions